hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/format/mor/MergeOnReadInputFormat.java

```diff
@@ -649,7 +656,8 @@ static class MergeIterator implements RecordIterator {
         int[] requiredPos,
         boolean emitDelete,
         int operationPos,
-        ParquetColumnarRowSplitReader reader) { // the reader should be with full schema
+        ParquetColumnarRowSplitReader reader, // the reader should be with full schema
+        String mergeClass) {
       this.tableSchema = tableSchema;
```

Comment:
Do we need to pass around the `mergeClass` explicitly ? You can fetch that through the `flinkConf` right ?